fe1-web/__tests__/model/network/FromJsonData.test.ts

```diff
@@ -44,6 +52,7 @@ describe('=== fromJsonData checks ===', () => {
   const time = STANDARD_TIMESTAMP;
   const name = 'poof';
   const location = 'Lausanne';
+  const mockVersion = STRINGS.election_version_identifier;
```

Comment:
I am not 100% sure what the Version field in the SetupElection should be, but having communicated with Victor, they just set it to '1.0.0', which I do here

Reply:
I think that's specified in the e-voting document :)